src/app/config/urls.ts

```diff
@@ -184,3 +184,8 @@ export const jobTypeToPath = (type: JobTypes, result?: boolean) => {
   }
   throw new Error(`"${type}"invalid job type`);
 };
+
+export const getHelpEntryPath = (entry: string) =>
```

Comment:
Wondering if it would be possible to overload the `getEntryPath` function above instead of defining a new one, either by specifying the type of namespace as `SearchableNamespace|Location` and then checking the type in the function (maybe risky), or by creating another `getEntryPathFromLocation` more generic method.

Reply:
Good idea, went with your last suggestion